sdk/resourcemanager/Proto.Client/compute/VirtualMachineScaleSetRollingUpgrade.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.ResourceManager.Core;
+
+#nullable disable
+
+namespace Proto.Compute
+{
+    /// <summary> A Class representing a VirtualMachineScaleSetRollingUpgrade along with the instance operations that can be performed on it. </summary>
+    public class VirtualMachineScaleSetRollingUpgrade : VirtualMachineScaleSetRollingUpgradeOperations
```

Comment:
Can we do a rename on this (both here and in autorest) to a RollingUpgrade?

The fact that it belongs to a VirtualMachineScaleSet will be obvious since the GetRollingUpgrade() method is on the VirtualMachineScaleSet object